common/withdraw_tx.c

```diff
@@ -29,11 +30,12 @@ struct bitcoin_tx *withdraw_tx(const tal_t *ctx,
 		const void *map[2];
 		map[0] = int2ptr(0);
 		map[1] = int2ptr(1);
-		bitcoin_tx_add_output(tx, scriptpubkey_p2wpkh(tx, changekey),
-				      change);
+		script = scriptpubkey_p2wpkh(tx, changekey);
```

Comment:
Commit 9ebb521c8524e52e294f7d1d68e5aeea65380ceb in #3006 and #2964 is a better solution for this.

Reply:
Removed!